src/libertem/io/dataset/seq.py

```diff
@@ -188,8 +191,10 @@ def _do_initialize(self):
         self._image_count = int(
             (self._filesize - self._image_offset) / header['true_image_size']
         )
-        if self._image_count != np.prod(self._scan_size):
-            raise DataSetException("scan_size doesn't match number of frames")
+        min_sync_offset = -1*np.prod(self._scan_size) + 1
+        max_sync_offset = np.prod(self._scan_size) - 1
+        if not min_sync_offset <= self._sync_offset <= max_sync_offset:
+            raise DataSetException("incorrect offset")
```

Comment:
Possibly should tell the user what is in correct about it, i.e. the range that it should be in